distribution/lib/Standard/Base/0.0.0-dev/src/Main.enso

```diff
@@ -121,12 +120,16 @@ import project.Network.HTTP.HTTP_Version.HTTP_Version
 import project.Network.URI.URI
 import project.Random
 
+from project.Data.Json.Extensions import all
```

Comment:
Was the original point of the Extensions files to speed up compilation/startup? Does this remove that benefit?

Reply:
What do you mean? I don't understand your question. In these `Extensions` modules, we have extension methods, and we need to import and export all these methods. I have recently changed the form of these import and export statements as there was a clash of `Extensions` symbol.